apps/gallery/utils.py

```diff
@@ -0,0 +1,33 @@
+from django.conf import settings
+from django.urls import reverse
+from django.utils.encoding import force_bytes
+from django.utils.http import urlsafe_base64_encode
+
+from gallery.enums import PrivateFileModuleType
+from deep.permissions import ProjectPermissions as PP
+from lead.models import Lead
+
+
+def get_private_file_url(private_file_module_type: PrivateFileModuleType, entry):
+    return '{protocol}://{domain}{url}'.format(
+        protocol=settings.HTTP_PROTOCOL,
+        domain=settings.DJANGO_API_HOST,
+        url=reverse(
+            'external_private_url',
+            kwargs={
+                'module': private_file_module_type.value,
+                'identifier': urlsafe_base64_encode(force_bytes(entry.id)),
+                'filename': entry.entry_attachment.file.name
+            }
+        )
+    )
+
+
+def check_private_entry_qs(user, project, level):
```

Comment:
We need to update the name of this function.

With something like this
```suggestion
def check_entry_confidential_level_permisison(user, project, level):
```